# Name of the property
string name
# Value of the property
string value
